src/js/components/addData/AddDataMeta.jsx

```diff
@@ -118,10 +127,42 @@ export default class AddDataMeta extends React.Component {
         }
     }
 
-    submitMetadata(){
-        this.props.updateMetaData(this.state);
-    }
-
+    submitMetadata(e){
+        let agency = this.state.agency, 
+            endDate = this.state.endDate,
+            dateType = this.state.dateType,
+            year = endDate.substr(3),
+            quarter = endDate.substr(0, 2);  
+        switch (quarter) {
+            case '12': quarter = '3';
```

Comment:
So December 2016 is quarter 3 of 2017? Why is quarter 3, 6, 9, 12 instead of 1, 2, 3, 4?